packages/panzer/disc-fe/src/evaluators/Panzer_ProjectToFaces_impl.hpp

```diff
@@ -106,6 +111,9 @@ ProjectToFaces(
     this->addEvaluatedField(gatherFieldNormals);
 
   } else {
+    TEUCHOS_ASSERT(quad_degree == 1); // One pt quadrature for fast method
+    TEUCHOS_ASSERT(std::strstr(basis->getCellTopology()->getBaseName(),"Hexahedron") != nullptr);
```

Comment:
What is this line doing?

Reply:
Enforcing that it can only be used for hex topology. There are multiple Hexahedron topologies that can be returned from shards: "Hexehedron_8", "Hexahedron_27", ...

Reply:
Thanks&mdash;I saw `strstr` and my mind went to `stringstream`.